src/build_vehicle_gui.cpp

```diff
@@ -1038,7 +1038,12 @@ void DrawEngineList(VehicleType type, const Rect &r, const GUIEngineList &eng_li
 		StringID str = hidden ? STR_HIDDEN_ENGINE_NAME : STR_ENGINE_NAME;
 		TextColour tc = (item.engine_id == selected_id) ? TC_WHITE : (TC_NO_SHADE | ((hidden | shaded) ? TC_GREY : TC_BLACK));
 
-		SetDParam(0, PackEngineNameDParam(item.engine_id, EngineNameContext::PurchaseList, item.indent));
+		if (show_count) {
+			// relies on show_count to find 'Vehicle in use' panel of autoreplace window
```

Comment:
Uh oh, more nitpicking :) Wrong comment style, should be `/* */`.

Reply:
Ternary operator?